README.md

```diff
@@ -76,21 +78,46 @@ docker build --target production --tag todo-app:prod .
 
 To run the container, run:
 ```
-docker run -p 8000:8000 --env-file ./.env todo-app:prod
+docker run -p 8000:8000 --env-file ./.env -it todo-app:prod
 ```
 
 Now visit [`http://localhost:8000/`](http://localhost:8000/) in your web browser to view the app.
 
-## Running the App in a VM
+### Running in a VM
 
 To Provision a VM from an Ansible Control Node:
 
 - SSH into the control node
-- Run `ansible-playbook my-ansible-playbook.yml -i my-ansible-inventory`
+- Run:
+```
+ansible-playbook my-ansible-playbook.yml -i my-ansible-inventory
+```
+
 - Visit `http://host.ip.address:5000/` in your browser to view the app
 
-## Running Tests
 
-To run the test suite in VS code, you need to set up the virtual environment. Press `Cmd/Ctrl + Shift + P` and select `Python: Select Interpreter`. Select the Python executable in the new `.venv` directory, which is `./.venv/Scripts/python.exe` on Windows.
+## Running the Tests
+
+### Running locally
+
+To run the tests (integration and unit), use the command 
+```
+poetry run pytest tests
+``` 
+in the DevOps-Course-Starter directory. This will search the "tests" directory for files starting in `test_` or ending in `_test`. Inside those files, any function starting with `test_` will be considered a test.
+
+### Running in a Docker Container
+
+To create the image, run the command:
+```
+docker build --target test --tag todo-app:test .
+```
+
+To run the tests, run:
+```
+docker run todo-app:test
+```
+
+### Running in a VM
 
```

Comment:
A nice addition to your docs would be a quick note that linked to the deployed website <http://frasto-todo-app.azurewebsites.net/>